client/pages/event.jsx

```diff
@@ -1,22 +1,52 @@
-
-import React, { useEffect, useState } from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
 import { useParams } from "react-router-dom";
 import formatDate from '../lib/format-date';
+import GuestList from '../components/guest-list';
+import { GoogleMap, useLoadScript, Marker } from '@react-google-maps/api';
+import usePlacesAutocomplete, { getGeocode, getLatLng } from 'use-places-autocomplete';
+import { Combobox, ComboboxInput, ComboboxPopover, ComboboxList, ComboboxOption } from '@reach/combobox';
 
-export default function Event( ) {
+export default function Event() {
   const [event, setEvent] = useState();
+  const [selected, setSelected] = useState(null);
   const { eventId } = useParams();
+  const libraries = useMemo(() => ['places'], []);
```

Comment:
I assume you learned to use `useMemo` from Google Maps documentation? I am guessing the purpose is to make sure that `libraries` is stable and doesn't change from render to render. I would guess that `useLoadScript` puts `libraries` into a dependency array somewhere, so it needs to be stable.

An alternative would be to declare `libraries` at the top level of the file, outside the `Event` function. I'm not saying that's better (`useMemo` is great here), just that it would probably also work.